shared/components/NextCommunityEvent/index.js

```diff
@@ -87,9 +93,16 @@ const dateTimeType = PropTypes.shape({
 
 NextCommunityEvent.propTypes = {
   title: PropTypes.string,
+  ticketsAvailable: PropTypes.bool,
```

Comment:
We should check this PropTypes, I don't think that they are all used. 
ticketsAvailable, ticketReleaseDate, externalLinks are not